apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +547,11 @@ var ConversationView = {
 
     var prevPanel = args.meta.prev;
 
+    var emailThread = Settings.supportEmailRecipient &&
+      Threads.active.participants.some(Utils.isEmailAddress);
+
+    Compose.setupLock({ getType: () => emailThread ? 'mms' : null });
```

Comment:
> it's fine to me;

Good!

> also I actually think we should return 'mms' for group mms threads...

I'm not sure if we can do this now, if we force ```mms``` for group thread, user will see mms label near composer input, but when user sends message, we'll actually send several SMSs, or I missing something?